Remotion/Core/Core/Configuration/ConfigurationElementCollection.cs

```diff
@@ -22,7 +22,7 @@ namespace Remotion.Configuration
 {
   public interface INamedConfigurationElement
   {
-    string Name { get; }
+    string? Name { get; }

```

Comment:
Was this an automated change? The 2 implementations of this interface are `string~`. This might be a bug in the rewriter. Please clarify this before I continue the review.